CRM/Financial/Form/FinancialTypeAccount.php

```diff
@@ -300,7 +300,9 @@ public function postProcess() {
         $params['entity_id'] = $this->_aid;
       }
       $financialTypeAccount = CRM_Financial_BAO_FinancialTypeAccount::add($params, $ids);
-      CRM_Core_Session::setStatus(ts('The financial type Account has been saved.'));
```

Comment:
So, what I was alluding to above is I think this could be
```
try {
  $financialTypeAccount = CRM_Financial_BAO_FinancialTypeAccount::add($params, $ids);
}
catch (CRM_Core_Exception $e) {
  CRM_Core_Session::statusBounce($e->getMessage());
}
```

There is still an issue in that validateRelationship takes a slightly formatted object - I'd be inclin